pkg/agent/poller/poller.go

```diff
@@ -61,7 +61,7 @@ func (m *pollExecutor) RegisterWatch(eventChan chan *proto.Event, errChan chan e
 		return
 	}
 
-	if m.sequence.GetSequence() == 0 {
+	if m.sequence.GetSequence() < 0 {
```

Comment:
Is there a concern to starting without a persistent cache? Would the sequence ID be 0 in this case?

Reply:
In both mode(gRPC or poll), if persistent cache is not enabled initCache is called(https://github.com/Axway/agent-sdk/blob/APIGOV-24414/pkg/agent/eventsync.go#L91) which places call to get events from harvester with sequence ID 0 (all events) which initializes the resource cache and also sets the current sequence to the cache(https://github.com/Axway/agent-sdk/blob/3493072c0edb217dec9834323aaf9655e63c74c1/pkg/agent/eventsync.go#L121).

So this should be ok